build/mobilenet-v1/build.py

```diff
@@ -144,12 +162,28 @@ def select_build_steps(platform):
             build_cfg.DataflowOutputType.ESTIMATE_REPORTS,
             build_cfg.DataflowOutputType.BITFILE,
             build_cfg.DataflowOutputType.DEPLOYMENT_PACKAGE,
+            build_cfg.DataflowOutputType.STITCHED_IP,
```

Comment:
To enable the stitched IP rtl simulation, we might need to add the create stitched IP step above.

Reply:
Resolved in [c3023fb](https://github.com/Xilinx/finn-examples/pull/109/commits/c3023fbd27e2710e9d3c4367d92c75653a04ee30)